Library/Formula/ld64.rb

```diff
@@ -27,6 +22,8 @@ class Ld64 < Formula
   depends_on "cctools-headers" => :build
   depends_on "dyld-headers" => :build
   depends_on "libunwind-headers" => :build
+  # No CommonCrypto
```

Comment:
It is actually able to link against the system libcrypto; I think the issue was that that user's build opportunistically picked up their Homebrew-provided openssl instead of the system one.

Reply:
understood, so is it possible to go the other way & explicitly link against the system libcrypto so that no matter what happens in Tigerbrew, ld will work.